person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(id, age, parent_permission: true, name: 'Unknown')
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
```

Comment:
- [ ] the ID should be created in the initialize method it should not be passed down, have a look at [this lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars) for more details.